### PR TITLE
SISRP-3683 Mock CS Awards feed and frontend connection

### PR DESCRIPTION
**Pull request by ctweney**
https://jira.berkeley.edu/browse/SISRP-3683

Some refactoring here: CampusSolutions now has a base Proxy class. Finaid is now a Merged model consisting of the old MyFinAid (messages from the old system) plus Awards (from the new CS system). MyAwards converts the CS XML format to something the front end can consume. 

The CS XML API lacks a few fields specified in the finaid_award.json dummy: Transaction IDs, Servicer, and Requirements. We'll have to figure out how to get those added to the feed. 

Note that the real CS API still isn't available, so this is all still mock data based on what Tamer made available on wikihub.

### CODE REVIEW

**Comment by ctweney** [comment]
@christianv should definitely see this one.

File: spec/models/campus_solutions/awards_spec.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+describe CampusSolutions::Awards do
+
+  context 'mock proxy' do
+    let(:oski_uid) { '61889' }
+    let(:oski_student_id) { 11667051 }
+    let(:fake_proxy) { CampusSolutions::Awards.new(user_id: oski_uid, fake: true) }
+    let(:feed) { fake_proxy.get[:feed] }
+
+    it 'returns JSON fixture data by default' do
+      p "feed = #{feed.inspect}"
```
Comment (raydavis) [inline spec/models/campus_solutions/awards_spec.rb]: Looks like one of those dang strays...

**Comment by raydavis** [comment]
Back-end looks good to me.

**Comment by christianv** [comment]
@ctweney awesome!

**Pull request merged**